src/main/java/org/gbif/api/model/common/search/SearchRequest.java

```diff
@@ -40,29 +36,12 @@
 @SuppressWarnings("unused")
 public class SearchRequest<P extends SearchParameter> extends PageableBase {
 
-  @Hidden
   private Map<P, Set<String>> parameters = new HashMap<>();
-
-  @Schema(
-    description = "Full-record query parameter."
-  )
   private String q;
-
-  @Schema(
-    description = "If highlighted search matches are requested."
-  )
   private boolean highlight;
-
-  @Hidden
   private boolean spellCheck;
```

Comment:
This needs checking, does it affect the OpenAPI documentation?

Reply:
This was a merge error. Fixed now